src/Views/Form-Themes/Sequoia/config.php

```diff
@@ -0,0 +1,127 @@
+					'desc'       => __( 'The button label for displaying the additional payment fields.', 'give' ),
+					'type'       => 'text_medium',
+					'attributes' => [
+						'placeholder' => __( 'Add Payment Information', 'give' ),
+					],
+				],
+			],
+		],
+		'donation_amount'     => [
+			'name'   => __( 'Donation Amount', 'give' ),
+			'desc'   => __( 'Step descruiption will show up here if any', 'give' ),
+			'fields' => [
+				[
+					'id'         => 'reveal_label',
+					'name'       => __( 'Continue Button', 'give' ),
+					'desc'       => __( 'The button label for displaying the additional payment fields.', 'give' ),
+					'type'       => 'text_medium',
+					'attributes' => [
+						'placeholder' => __( 'Add Payment Information', 'give' ),
+					],
+				],
+			],
+		],
+		'payment_information' => [
+			'name'   => __( 'Payment Information', 'give' ),
+			'desc'   => __( 'Step description will show up here if any', 'give' ),
+			'fields' => [
+				[
+					'id'         => 'checkout_label',
+					'name'       => __( 'Submit Button', 'give' ),
+					'desc'       => __( 'The button label for completing a donation.', 'give' ),
+					'type'       => 'text_medium',
+					'attributes' => [
+						'placeholder' => __( 'Donate Now', 'give' ),
+					],
+				],
+			],
+		],
+		'thank-you'           => [
+			'name'   => __( 'Thank You', 'give' ),
+			'desc'   => __( 'Step description will show up here if any', 'give' ),
+			'fields' => [
+				[
+					'id'         => 'headline',
+					'name'       => __( 'Headline', 'give' ),
+					'desc'       => __( 'This message should be short and sweet. Make the donor feel good about their donation so they continue to give in the future. This text is required and you may use any of the available template tags within this message.', 'give' ),
+					'type'       => 'text',
+					'attributes' => [
+						'placeholder' => __( 'A great big thank you!', 'give' ),
+					],
+				],
+				[
+					'id'         => 'description',
+					'name'       => __( 'Description', 'give' ),
+					'desc'       => __( 'The description is displayed directly below the main headline and should be 1-2 sentences for best performance. You may use any of the available template tags within this message.', 'give' ),
+					'type'       => 'textarea',
+					'attributes' => [
+						'placeholder' => __( '{name}, you contribution means a lot and will be put to good use making a difference. We’ve sent your donation receipt to {donor_email}. ', 'give' ),
+					],
+				],
+				[
+					'id'         => 'image',
+					'name'       => __( 'Image', 'give' ),
+					'desc'       => __( 'Upload an eye-catching image that reflects your cause. The image is required and if none is provided the featured image will be a the fallback. If none is set you will see a placeholder image displayed on the form. For best results use an image that’s 600x400 pixels.', 'give' ),
+					'type'       => 'file',
+					'query_args' => [
```

Comment:
@henryholtgeerts You can remove this param. it is not supported by the `file` or `media` field. We are handling that internally.
ref: https://github.com/impress-org/givewp/blob/54151c5f09f3145e17a0fdb0558c2c859be80fd3/includes/admin/give-metabox-functions.php#L885

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/91bbb4a8e9265e3fca9cb6e01c2361e80d427480